inc/checks.php

```diff
@@ -34,68 +34,59 @@ function ns_theme_check_do_sniff( $theme_slug, $args = array() ) {
 
 	$args = wp_parse_args( $args, $defaults );
 
-	// Path to WordPress Theme coding standard.
-	PHP_CodeSniffer::setConfigData( 'installed_paths', NS_THEME_CHECK_DIR . '/vendor/wp-coding-standards/wpcs/', true );
-	PHP_CodeSniffer::setConfigData( 'csslint_path', NS_THEME_CHECK_DIR . '/node_modules/csslint/dist/cli.js --errors=errors', true );
+	// Set CLI arguments.
+	$cli_args  = get_theme_root() . '/' . $theme_slug;
```

Comment:
Did it https://github.com/dingo-d/ns-theme-check/commit/448b314f126f9e63025dca28acf3ca7c346e9c22

Do you want me to make a pull request?